### PR TITLE
Solarium 3.x: Fix for #618 DateTime is not set to UTC when using Extract in v3.x

### PR DESCRIPTION
**Pull request by jeroenherczeg**
Fix for https://github.com/solariumphp/solarium/issues/618

### CODE REVIEW

**Comment by jeroenherczeg** [comment]
Should I worry about styleci?

File: tests/Solarium/Tests/QueryType/Extract/RequestBuilderTest.php [hunk 1]
```diff
@@ -139,4 +139,26 @@ public function testContentTypeHeader()
         $this->assertEquals($headers,
                             $request->getHeaders());
     }
+
+    public function testDocumentDateTimeField()
+    {
+        $timezone = new \DateTimeZone('Europe/London');
+        $date = new \DateTime('2013-01-15 14:41:58', $timezone);
+        $document = $this->query->createDocument(array('date' => $date));
```
Comment (thePanz) [inline tests/Solarium/Tests/QueryType/Extract/RequestBuilderTest.php]: Aren't we using the short-form for arrays? (I thought it was covered by the php-cs-styles) :/

**Comment by jeroenherczeg** [comment]
In 3x branch, composer.json says this package supports PHP 5.3.9, I think short form array syntax was introduced in 5.4

**Comment by thePanz** [comment]
@jeroenherczeg true. I am quite scared about this library still supporting 5.3 (see: http://php.net/supported-versions.php). should we move to 5.6 (and deprecate this library's v3 after php 5.6 is out of life? wdyt?)

**Comment by jeroenherczeg** [comment]
Moving to 5.6 for v3 would be a breaking change not in a major version, could introduce a problem for someone, but then again still running 5.3 is also a problem. 

Deprecating v3 at the same time with 5.6, sounds good to me.

**Comment by jeroenherczeg** [comment]
Is it possible to merge this PR, or does it require changes?

**Comment by Schnaaf** [comment]
Can this be reviewed please?

**Comment by jeroenherczeg** [comment]
@thePanz Is there something blocking this PR?

**Comment by mkalkbrenner** [comment]
Since PHP 5.x and 7.0 and Solr versions before 6.4 are EOL we declare EOL for solarium 3.x, too.
For sure it is still available but we won't merge PRs anymore and there won't be another 3.x release.

**Pull request closed without merging**